server/auth/auth.go

```diff
@@ -337,20 +337,20 @@ func (u user) Type() string {
 
 // CanAccess determines if a User can access an object represented by the ACL
 // with a certain AccessType. It compares LDAP username and LDAP group.
-func (u user) CanAccess(acl knox.ACL, t knox.AccessType) bool {
+func (u user) CanAccess(acl knox.ACL, t knox.AccessType) (bool, string) {
 	for _, a := range acl {
 		switch a.Type {
 		case knox.User:
 			if a.ID == u.ID && a.AccessType.CanAccess(t) {
-				return true
+				return true, "0u" + u.ID
 			}
 		case knox.UserGroup:
 			if u.inGroup(a.ID) && a.AccessType.CanAccess(t) {
-				return true
+				return true, "0g" + a.ID
 			}
```

Comment:
we could probably add a `String()` function on the ACL entry type that would encode the string in this way.